### PR TITLE
Nicer shift masks

### PR DESCRIPTION
**Pull request by jfbastien**

### CODE REVIEW

**Comment by dschuff** [comment]
I think this is actually less nice.... :(

**Comment by jfbastien** [comment]
> I think this is actually less nice.... :(

But is it thirty-two less one nice, or sixty-four less one nice?

**Comment by dschuff** [comment]
its `nice less foo.txt`

**Comment by binji** [comment]
@dschuff wrap it in a macro

**Comment by kripken** [comment]
I think this is clearer personally, but if more people think otherwise I'm fine to leave it as is.

Instead, perhaps a comment "we don't shift more than the actual number of bits" would be just as good, with the previous code.

**Comment by jfbastien** [comment]
> I think this is clearer personally, but if more people think otherwise I'm fine to leave it as is.
> 
> Instead, perhaps a comment "we don't shift more than the actual number of bits" would be just as good, with the previous code.

Updated, is this better?

File: src/wasm.h [hunk 1]
```diff
@@ -165,7 +165,13 @@ class Literal {
     int64_t i64;
   };
 
-public:
+  // The RHS of shl/shru/shrs must be masked by bidwidth.
```
Comment (kripken) [inline src/wasm.h]: *bitwidth

otherwise lgtm

**Pull request merged**